docs/product-editor-development/product-editor-extensibility-guidelines.md

```diff
@@ -0,0 +1,236 @@
+# Product Editor Extensibility Guidelines
+
+> ⚠️ **Notice:** These guidelines are currently a **work in progress**. Please be aware that some details might be incomplete or subject to change. We appreciate your patience and welcome any contributions!
+
+Changelog:
+
+- 2021-11-17: Initial version
```

Comment:
Is that correct? That long ago?

Reply:
Hahaha... yes, we've been sitting on this for a few years... oops!!! Thanks for catching this!